app/components/pages/SubmissionWizard/steps/Files/index.js

```diff
@@ -190,27 +197,43 @@ const FilesPageContainer = ({
                   {uploadSupportFiles => (
                     <Mutation mutation={DELETE_SUPPORTING_FILES_MUTATION}>
                       {removeSupportFiles => (
-                        <SupportingUpload
-                          data-test-id="supportingFilesUpload"
-                          files={submissionFiles.filter(
-                            file => file.type === SUPPORTING_FILE,
+                        <Mutation mutation={DELETE_SINGLE_SUPPORTING_FILE}>
```

Comment:
I think we're better off renaming to the following:
`DELETE_ALL_SUPPORTING_FILES_MUTATION`
`DELETE_SUPPORTING_FILE_MUTATION`